src/ceph-disk/ceph_disk/main.py

```diff
@@ -2362,17 +2362,18 @@ def create_key(self):
         cluster = self.args.cluster
         bootstrap = self.args.prepare_key_template.format(cluster=cluster,
                                                           statedir=STATEDIR)
-        command_check_call(
+        command_with_stdin(
```

Comment:
@badone If the commit is based on a single commit from master, it should be done as a cherry-pick. Manual cherry-picking is not preferred, but if you think that's the optimal way to do it in this case, then add a line like this:

```
(manual cherry-pick of ...)
```